### DISCUSSION TITLE
More looking at commercial drones

### DISCUSSION THREAD

**Discussion by harrylerman** [dannyjacobs/ECHO#50] [General]
More digging into commercial drone solutions (punchline: I still think building our own is the way to go!)

The reality seems to be that all roads lead back to DJI, either the Matrice 210 or 300. The Arctic drone group I read about was using a 210. [The 300 costs $12,500 USD](https://store.dji.com/ca/product/matrice-300-rtk-and-dji-care-plus?vid=111261), and DJI doesn't make the 210 any more but on [Ebay they seem to go for about $7,000 USD](https://www.ebay.com/sch/i.html?_from=R40&_trksid=p2380057.m570.l1313&_nkw=matrice+210+rtk&_sacat=0). These drones would be configurable to do what we want, in a slightly clunky way - from what I've read, people use a 3rd party open source app to make flightplans and then get the waypoints on the drone in a kind of hackish way. Hard to say if it's any clunkier than using QGC.

The real issue with DJI (besides price) is reparability - there's only one authorized repair shop in the US/Canada and it's in Los Angeles. We could probably do some repairs ourselves as needed, but that gets into warranty issues etc too. But for other repairs we'd be looking at sending it to DJI.

Then there is the Yuneec drone we looked at before. They did come out with their RTK version. [It costs $5,300 USD and is out of stock](https://store.yuneec.com/yuneec-h520e-rtk-commercial-hexacopter-airframe-only.html) right now (unsure when it will be in stock, there are some rumblings on Reddit that Yuneec is having problems with the drone and is doing bug fixes - I don't know if it ever really was available in the first place). But I think @dannyjacobs did some testing with their non-RTK version and didn't like it. The drone has that huge bulky touch-screen controller whose battery lasts like 25 minutes if I remember, people hack their own battery packs into it, also a negative. Same comments on reparability, @dannyjacobs review was that this drone was pretty flimsy and broke easily, really designed to be almost disposable for companies doing survey work.

There are a few US companies making drones as well but in general they are very expensive. I looked at three that seem to get mentioned online a lot, [Inspired Flight](https://inspiredflight.com/), [Autel](https://auteldrones.com/), and [Kaizen Aero](https://kaizen.aero/). They're either very expensive like $20k-$30k and up, or are tiny and despite being RTK I doubt they could carry any payload. US drones seem to be expensive because of the harsh requirements coming out for privacy (DJI is banned for many applications because data goes back to their servers in China) and position reporting that is soon to be mandatory.

I really tried to be non-biased in looking into this. But I do think building our own stuff is the way to go. Happy to take any inputs! 

(also one of those US companies charges crazy prices for their drones but uses our same open-source Pixhawk flight controller, thought that was interesting)

**Comment by eegan** [comment]
You may be biased but what you say sounds reasonable. At very least, it's not a terrible idea to build our own!

**Comment by newburgh** [comment]
We've been looking at this outfit, which builds custom drones:
https://uavsystemsinternational.com/
So far, I wouldn't recommend them because I am trying to give the money and they haven't responded, maybe because I'm asking for some 'bespoke' things. However, their drones are more like 5-10k range, they have a decent set of customizations, and at least sell spare parts.

Curious if people have found other outfits like this which they don't like/do like?

**Comment by dannyjacobs** [comment]
I haven't shopped recently. This looks like a startup, they're probably hunting around for their market and price point. I hope they make it! But their margins are pretty thin.  I like that you can select your flight computer and even firmware.

I'd want to be able to specify the aux/computer power supply. Most of them are pretty noisy.  Or if changing the PS is a real hardship maybe one could take their loaners and do some testing. 

Also:
Hire English majors! They still exist!
<img width="650" alt="image" src="https://user-images.githubusercontent.com/63191/191026577-1f43c690-692e-4b8b-be9a-e660e25f9de2.png">
